fn_spamhaus_query/fn_spamhaus_query/util/config.py

```diff
@@ -0,0 +1,21 @@
+# (c) Copyright IBM Corp. 2010, 2019. All Rights Reserved.
+# -*- coding: utf-8 -*-
+
+"""Generate a default configuration-file section for fn_spamhaus_query"""
+
+from __future__ import print_function
+
+
+def config_section_data():
+    """Produce the default configuration section for app.config,
+       when called by `resilient-circuits config [-c|-u]`
+    """
+    config_data = u"""[fn_spamhaus_query]
+# The API endpoint URL to query Spamhaus Web Query Service
+spamhaus_wqs_url = https://apibl.spamhaus.net/lookup/v1/{}/{}
```

Comment:
* We should not have the user put {} in the app.config file so we should remove them here
* The placeholder should be: 
```
spamhaus_wqs_url = https://apibl.spamhaus.net/lookup/v1/
```

Reply:
ok